RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -21,7 +21,7 @@
 tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
     layerList = cms.vstring(
     #TOB
-    'TOB1+TOB2+MTOB3',
+    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
```

Comment:
remind me, please what "M" is for in MTOB4?

what's the effect of this on timing, efficiency and fake rates?
If there is any, please decouple changes in DQM plots from physics changes.